tests/setup.py

```diff
@@ -240,5 +221,7 @@ def tearDown(self) -> 'Generator':
 
     def doCleanups(self) -> 'Generator':
         # restore the user's configs
-        client_configs.update_configs()
+        close_test_view(self.view)
```

Comment:
Now `close_test_view` is called both here and from `await_boilerplate_end()`. Maybe latter not needed?

Reply:
Well, if the test fails then we still want to close the view, otherwise there will be a "dangling" view that was opened during a test.

Reply:
Right, so in `await_boilerplate_end` it would not be needed :)